app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MapFragment.java

```diff
@@ -176,10 +172,25 @@ public void onInfoWindowClick(Marker marker) {
                 AnalyticsUtil.selectContent(getContext(), "Map");
             }
         });
-        //night mode map
-        //MapStyleOptions style = MapStyleOptions.loadRawResourceStyle(getContext(),R.raw.mapstylenight);
-        //googleMap.setMapStyle(style);
-        googleMap.setMapType(GoogleMap.MAP_TYPE_NORMAL);
+        //style mode mode map
+        if (WhibApp.getInstance().getThemeId()== R.style.AppTheme_Dark){
+            MapStyleOptions style = MapStyleOptions.loadRawResourceStyle(getContext(),R.raw.mapstylenight);
+            googleMap.setMapStyle(style);
+        }else if(WhibApp.getInstance().getThemeId()== R.style.AppTheme_Dark_Purple){
+            MapStyleOptions style = MapStyleOptions.loadRawResourceStyle(getContext(),R.raw.mapstyleaubergine);
+            googleMap.setMapStyle(style);
+
+        }else if(WhibApp.getInstance().getThemeId()== R.style.AppTheme_Dark_Red){
+            MapStyleOptions style = MapStyleOptions.loadRawResourceStyle(getContext(),R.raw.mapstylered);
+            googleMap.setMapStyle(style);
+
+        }
+        else {
```

Comment:
This is missing the new `AppTheme_Dark_Green`

Reply:
Also it does not make sure that new dark themes automatically get some default dark map.